### ISSUE TITLE
DataStore SSL cert

### ISSUE THREAD

**Issue by arashm2** [Esri/arcgis-cookbook#251]
Hello,

The Chef recipes currently don't have a way to update the SSL cert for DataStore, correct?
https://enterprise.arcgis.com/en/data-store/latest/install/windows/update-ads-ssl-certificate.htm

If so, could this be a feature request?

Thanks

**Comment by cameronkroeker** [comment]
@arashm2 This is correct, however we will evaluate this feature and see if it can be included in a future release.
